src/tests/system/tests/test_sss_override.py

```diff
@@ -216,16 +211,20 @@ def test_sss_overrides__export_then_import_user_and_group_override_data(client:
     group.delete()
 
     client.sssd.restart()
-    assert not client.sss_override.user("user1").get()
-    assert not client.sss_override.group("group1").get()
+    assert not client.sss_override.user("user1").get(), "Local override data exists!"
+    assert not client.sss_override.group("group1").get(), "Local override data exists!"
     client.sss_override.import_data()
     client.sssd.restart()
 
-    assert client.sss_override.user("user1").get(["name"]) == {"name": ["o-user1"]}
-    assert client.sss_override.group("group1").get(["name"]) == {"name": ["o-group1"]}
+    assert client.sss_override.user("user1").get(["name"]) == {
+        "name": ["o-user1"]
+    }, "Local override data does not exist!"
+    assert client.sss_override.group("group1").get(["name"]) == {
+        "name": ["o-group1"]
+    }, "Local override data does not exist!"
```

Comment:
It would be great to check that no extra rule was imported, but I'm no sure there is a way to do that.

Reply:
I added an ldbsearch and ensures that no objects are found.

Reply:
```
collected 560 items / 559 deselected / 1 selected

tests/test_sss_override.py::test_sss_overrides__root_uid_gid_cannot_be_used (ldap) 

====================== 1 passed, 559 deselected in 10.83s ======================
PASSED [100%]
```